### PR TITLE
Split UIImages to their respective commands

### PR DESCRIPTION
**Pull request by openbots-ff**

### CODE REVIEW

**Review by openbots-ch — APPROVED** [review]

File: OpenBots.Studio/UI/CustomControls/CustomUIControls/UIImage.cs [hunk 1]
```diff
@@ -1,237 +1,23 @@
...
-        public static ImageList UIImageList()
-        {
-            Dictionary<string, Image> imageIcons = UIImageDictionary();
             ImageList uiImages = new ImageList();
             uiImages.ImageSize = new Size(18, 18);
 
-            foreach (var icon in imageIcons)
+            foreach (var icon in commandIcons)
             {
                 //var someImage = icon.Value;
```
Comment (openbots-ab) [inline OpenBots.Studio/UI/CustomControls/CustomUIControls/UIImage.cs:22]: Please remove the comments

File: OpenBots.Studio/UI/Forms/ScriptBuilder Forms/frmScriptBuilder.cs [hunk 2]
```diff
@@ -260,10 +260,7 @@ private void frmScriptBuilder_Load(object sender, EventArgs e)
                 _scriptVariables = new List<ScriptVariable>();
                 _scriptElements = new List<ScriptElement>();
             }
-            //pnlHeader.BackColor = Color.FromArgb(255, 214, 88);
-
-            //instantiate and populate display icons for commands
-            _uiImages = UIImage.UIImageList();
+            //pnlHeader.BackColor = Color.FromArgb(255, 214, 88);
```
Comment (openbots-ab) [inline OpenBots.Studio/UI/Forms/ScriptBuilder Forms/frmScriptBuilder.cs:263]: Please remove the comment

File: OpenBots.Commands/OpenBots.Commands.API/ExecuteDLLCommand.cs [hunk 3]
```diff
@@ -70,6 +70,7 @@ public ExecuteDLLCommand()
 		{
 			CommandName = "ExecuteDLLCommand";
 			SelectionName = "Execute DLL";
+			CommandIcon = Core.Properties.Resources.command_run_code;
```
Comment (openbots-ab) [inline OpenBots.Commands/OpenBots.Commands.API/ExecuteDLLCommand.cs:73]: Please remove the extra namespaces **Core.Properties**

**Review by openbots-ab — CHANGES_REQUESTED** [review]
Please make few minor changes

File: OpenBots.Studio/UI/CustomControls/CustomUIControls/UIImage.cs (hunk 1 above)
Comment (openbots-ff) [reply OpenBots.Studio/UI/CustomControls/CustomUIControls/UIImage.cs:22]: those comments were already there. They're related to the icon rendering and I may still revisit/use them at some point.

File: OpenBots.Studio/UI/Forms/ScriptBuilder Forms/frmScriptBuilder.cs (hunk 2 above)
Comment (openbots-ff) [reply OpenBots.Studio/UI/Forms/ScriptBuilder Forms/frmScriptBuilder.cs:263]: same as above

File: OpenBots.Commands/OpenBots.Commands.API/ExecuteDLLCommand.cs (hunk 3 above)
Comment (openbots-ff) [reply OpenBots.Commands/OpenBots.Commands.API/ExecuteDLLCommand.cs]: done

File: OpenBots.Studio/UI/CustomControls/CustomUIControls/UIImage.cs (hunk 1 above)
Comment (openbots-ab) [reply OpenBots.Studio/UI/CustomControls/CustomUIControls/UIImage.cs:22]: It's fine.

**Review by openbots-ab — APPROVED** [review]
Looks good to me

File: OpenBots.Commands/OpenBots.Commands.Window/MoveWindowCommand.cs [hunk 4]
```diff
@@ -49,7 +50,9 @@ public MoveWindowCommand()
 			CommandName = "MoveWindowCommand";
 			SelectionName = "Move Window";
 			CommandEnabled = true;
-			
+			CommandIcon = Resources.command_window;
+
+
```
Comment (openbots-nc) [inline OpenBots.Commands/OpenBots.Commands.Window/MoveWindowCommand.cs:55]: There is an extra line here

File: OpenBots.Commands/OpenBots.Commands.Window/SetWindowStateCommand.cs [hunk 5]
```diff
@@ -41,7 +42,9 @@ public SetWindowStateCommand()
 			CommandName = "SetWindowStateCommand";
 			SelectionName = "Set Window State";
 			CommandEnabled = true;
-			
+			CommandIcon = Resources.command_window;
+
+
```
Comment (openbots-nc) [inline OpenBots.Commands/OpenBots.Commands.Window/SetWindowStateCommand.cs:47]: There is an extra blank line here

File: OpenBots.Commands/OpenBots.Commands.Window/WaitForWindowToExistCommand.cs [hunk 6]
```diff
@@ -40,7 +41,9 @@ public WaitForWindowToExistCommand()
 			CommandName = "WaitForWindowToExistCommand";
 			SelectionName = "Wait For Window To Exist";
 			CommandEnabled = true;
-			
+			CommandIcon = Resources.command_window;
+
+
```
Comment (openbots-nc) [inline OpenBots.Commands/OpenBots.Commands.Window/WaitForWindowToExistCommand.cs:46]: There is an extra blank line here

**Review by openbots-nc — APPROVED** [review]

**Pull request merged**